packages/rancher-logging/rancher-logging.patch

```diff
@@ -87,4 +87,6 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +    tag: 0.1.0
 +
 +global:
-+  systemDefaultRegistry: ""
++  cattle:
++    systemDefaultRegistry: ""
```

Comment:
nit: add new line

Reply:
This will more than likely collide with another pr, I will fix it in there.